lib/trento_web/controllers/v1/settings_controller.ex

```diff
@@ -147,6 +147,44 @@ defmodule TrentoWeb.V1.SettingsController do
     end
   end
 
+  operation :get_analytics_settings,
+    summary: "Fetches the Analytics settings",
+    tags: ["Platform"],
+    responses: [
+      ok:
+        {"Analytics settings fetched successfully", "application/json",
+         Schema.Platform.AnalyticsSettings},
+      not_found: Schema.NotFound.response()
+    ]
+
+  def get_analytics_settings(conn, _) do
+    case Settings.get_analytics_settings() do
+      {:ok, settings} -> render(conn, :analytics_settings, %{settings: settings})
+      {:error, :analytics_settings_not_configured} -> {:error, :not_found}
```

Comment:
Just put here the same error you put in the `settings.ex` `get_analytics_settings` function

Reply:
I ended up creating an `analytics_settings_not_configured ` in the Fallback Controller because the `settings_not_found` refers to SUMA settings only.

Reply:
But you are not using it. You are returning `{:error, :not_found}` here, so the entry you added in the fallback controller is not going to be used